### PR TITLE
Story/VSPC-88

### PR DESCRIPTION
**Pull request by prjadhav14**

### CODE REVIEW

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
the links are way off on the staff side now

**Comment by jdamerow** [comment]
or on the public page, not sure, but they are very different.

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp [hunk 1]
```diff
@@ -1051,7 +1051,7 @@ Delete Space
 
 <c:if test="${not empty space.image}">
 	<div id="space">
-		<img id="bgImage" width="800px" src="<c:url value="/api/image/${space.image.id}" />" />
+		<img style="min-height: 500px; min-width:800px;" id="bgImage" width="800px" src="<c:url value="/api/image/${space.image.id}" />" />
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp:1054]: that seems wrong. if the image is smalelr than 800px we don't want to blow it up

File: vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp [hunk 2]
```diff
@@ -188,8 +187,8 @@ function drawLinks() {
             </div>
             <i class="fas fa-bars fa-lg barPosition" onclick="openNav()"></i>
         </div>
-        <div class="spaceClass"
-            style="width: ${display.width}px; height: ${display.height}px; margin: auto; background-size: cover; border-radius: 13px;">
+        <div class="spaceClass" style="width: 800px; margin: auto; display: flex;">
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp:190]: if an image is smaller than 800px, we don't want to blow it up.

File: vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp [hunk 3]
```diff
@@ -86,6 +86,8 @@ function drawLinks() {
     {
         var posX = parseInt($("#space").css('margin-left')) + $("#space").position().left; 
         var posY = $("#space").position().top;
+        console.log(posX);
+        console.log(posY);
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp:90]: remove

File: vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp [hunk 4]
```diff
@@ -114,6 +116,8 @@ function drawLinks() {
...
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp:120]: remove

**Comment by prjadhav14** [comment]
In the old layout for public profile min-height parameter was used for space image and hence the image used to get stretched if less than 500px. I think this was the reason links appeared at different positions on staff and public profile if images height is less than 500px. In the latest commit, a fix has been applied such that stored display height and width of the image is retrieved while loading the image at both staff and public site.

**Pull request merged**